experimental/builder/include/ck_tile/builder/testing/testing.hpp

```diff
@@ -205,6 +205,20 @@ template <auto SIGNATURE>
     requires ValidUniqueInputs<SIGNATURE>
 UniqueInputs<SIGNATURE> alloc_inputs(const Args<SIGNATURE>& args);
 
+/// @brief Allocate inputs corresponding to a signature.
+///
+/// The `init_inputs()` function is used to initialize pseudo-random data
+/// to the tensors specified in the Inputs structure.
+///
+/// @tparam SIGNATURE the signature to specialize the structure for.
+///
+/// @see Inputs
+/// @see UniqueInputs
+/// @see tensor_initialization
+template <auto SIGNATURE>
+    requires ValidUniqueInputs<SIGNATURE>
+void init_inputs(const Args<SIGNATURE>& args, UniqueInputs<SIGNATURE>& inputs);
```

Comment:
This does not need to take ownership
```suggestion
void init_inputs(const Args<SIGNATURE>& args, Inputs<SIGNATURE>& inputs);
```